pkg/api/types.go

```diff
@@ -455,6 +455,9 @@ type PersistentVolumeSpec struct {
 	// means that this volume does not belong to any StorageClass.
 	// +optional
 	StorageClassName string
+	// A comma-separated list of mount options. Not validated--mount will simply fail if one is invalid.
+	// +optional
+	MountOptions *string
```

Comment:
From the discussion https://github.com/kubernetes/community/pull/321#discussion_r99095392, it seems people had no strong objections to a comma-separated string once we decided to give up per-option validation, so it made it into the final proposal. I guess that's not a very good justification.

The only other reason I can think of is: since StorageClass.parameters is restricted to map[string]string, there is some consistency in making persistentVolume.spec.mountOptions a string like storageClass.parameters[mountOptions] is.